vumi/persist/txredis_manager.py

```diff
@@ -154,6 +174,15 @@ def ttl(self, key):
         d.addCallback(lambda r: (None if r < 0 else r))
         return d
 
+    def persist(self, key):
+        self._send('PERSIST', key)
+        return self.getResponse()
```

Comment:
Does txredis not implement this? If not, could we add a docstring mentioning that like we do above?